src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.cs

```diff
@@ -139,10 +139,10 @@ internal static string GetNormalizedStrongNameHash(AssemblyName name)
                     bw.Write(publicKey);
                     bw.Write(name.Version.Major);
                     bw.Write(name.Name);
-                }
 
-                ms.Position = 0;
-                return GetStrongHashSuitableForObjectName(ms);
```

Comment:
I see.  And BinaryWriter auto-flushes to the underlying stream on every write (or doesn't have a buffer at all) so you don't need to add flush calls before reading from the MemoryStream?